src/app/core/configurable-enum/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -230,48 +239,46 @@ export class BasicAutocompleteComponent<O, V = O>
   }
 
   onFocusIn() {
```

Comment:
can we rename this function to describe what it does rather than when it is triggered (especially as this is not just an event handler but also explicitly called programmatically). Maybe activeateAutocompleteMode or resetAutocompleteMode or something?